infrastructure/README.md

```diff
@@ -0,0 +1,11 @@
+# Infrastructure
+
+This folder contains the infrastructure-as-code (using Terraform) for this project.
+
+## Pipelines
+
+There are two pipelines, one for static checks during PR, and only to plan and apply the infrastructure. These are based on common-pipeline-templates.
```

Comment:
..., and **one** to plan and apply the infra...

Reply:
Good spot!